presto-hive/src/main/java/com/facebook/presto/hive/HiveTableLayoutHandle.java

```diff
@@ -105,11 +117,23 @@ public SchemaTableName getSchemaTableName()
     }
 
     @JsonProperty
-    public List<ColumnHandle> getPartitionColumns()
+    public List<HiveColumnHandle> getPartitionColumns()
```

Comment:
Actually no, it is needed. I changed it to `HiveColumnHandle` to avoid downcasting here: https://github.com/prestodb/presto/pull/13453/files#diff-970630f02fcbabdd32239e406511230bR301